templates/base.tpl

```diff
@@ -1,16 +1,14 @@
 <!doctype html>
```

Comment:
I would prefer to save this for a followup change, but I wholeheartedly agree with you that we should port them to Soy.